Card/card.css

```diff
@@ -0,0 +1,180 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+}
+
+:root {
+    --white: white;
+    --black: black;
+    --hover-color: rgb(247, 242, 242);
+    --grey: grey;
+    --gainsboro: gainsboro;
+}
+
+.hero-container1,
+.hero-container2,
+.hero-container3,
+.hero-container4 {
+    max-width: 23rem;
+    margin: auto;
+    background-color: var(--white);
+    border-radius: 1rem;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+    border: 2px solid black;
+}
+
+.container1:hover {
+    background-color: var(--hover-color);
+}
+
+.hero-container1 .container1,
+.hero-container1 .container2,
+.hero-container2 .container2 {
+    display: flex;
+    gap: 20px;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem;
+    border-radius: 1rem;
+}
+
+.hero-img {
+    width: 5 rem;
+    height: 10rem;
+    border-top-left-radius: 1rem;
+    border-bottom-right-radius: 1rem;
+    padding-top: 0.5rem;
+}
+
+.hero-heading {
+    margin-bottom: 0px;
+    padding-bottom: 0px;
+}
+
+h6 {
+    padding-top: 0px;
+    margin-top: 0px;
+    text-align: center;
```

Comment:
I think for heading tags you can make `margin` and `padding` `0` in reset.css file. It will help you throughout your design. And it will also add consistency.